### DISCUSSION TITLE
RE: auvisio S06 IR Controller compatibility documentation

### DISCUSSION THREAD

**Discussion by DigiH** [arendst/Tasmota#18816] [Q&A]
Re: the model compatibility difference, as stated on
https://templates.blakadder.com/auvisio_S06.html

I just ordered an auvisio S06 IR Controller from Pearl, making sure it was the URC-150.app (NX-4519-905). Unfortunately this model now also has the BK7231N CBU and not the compatible TYWE3S any longer.

There is however another product URC-150.app NX-4519-953, with an identical looking product page. So possibly someone else might want to try that one.

Either way, you might want to update the docs to save others from ordering and having to return.

**Comment by sfromis** [comment]
Hit the edit link on the right of the page, and submit an update suggestion.
